tools/bridge/end2end-tests/tests/test_transfer.py

```diff
@@ -109,13 +117,16 @@ def __init__(
 
     def start(self):
         """Starts the service and wait for it to be up """
-        self.process = Popen(self.args, env=self.env, **self._process_settings)
-        try:
-            self._wait_for_up()
-            return self.process
-        except TimeoutException:
-            self.terminate()
-            raise
+        if self.is_up():
```

Comment:
This check only checks if the service is up. This is not the same as started, so please remove it here.

Reply:
Is it not? Since `start()` will wait for the service to be up before returning. If we do no have this and a tests attempt to start a bridge twice, the bridges fixture will not properly terminate all bridges.

Reply:
As discussed, this is changed for a is_started check